src/main/java/gg/solarmc/kitpvp/stat/PlaceholderAPIHook.java

```diff
@@ -0,0 +1,49 @@
+package gg.solarmc.kitpvp.stat;
+
+import gg.solarmc.kitpvp.KitpvpPlugin;
+import gg.solarmc.loader.kitpvp.KitPvpKey;
+import gg.solarmc.loader.kitpvp.OnlineKitPvp;
+import me.clip.placeholderapi.expansion.PlaceholderExpansion;
+import org.bukkit.entity.Player;
+
+public class PlaceholderAPIHook extends PlaceholderExpansion {
+
+    private final KitpvpPlugin plugin;
+
+    public PlaceholderAPIHook(KitpvpPlugin plugin) {
+        this.plugin = plugin;
+    }
+
+    @Override
+    public String getIdentifier() {
+        return "kitpvp";
+    }
+
+    @Override
+    public String getAuthor() {
+        return "Aurium";
+    }
+
+    @Override
+    public String getVersion() {
+        return "0.1.0";
+    }
+
+    @Override
+    public String onPlaceholderRequest(Player player, String identifier){
+
+        if(player == null) { return ""; }
```

Comment:
I don't know what PAPI's contract on nullability of the player is, but generally, I suggest a fail-fast approach of handling nulls. If you receive a null value which shouldn't be null, just let the code fail. The NPE will show what went wrong.

Reply:
This was the suggested approach for Placeholder API, and i believe it regularily uses nullity as an argument rather than an indicator of something gone wrong